res/schema.xml

```diff
@@ -447,4 +447,37 @@ METADATA
       ALTER TABLE cues ADD COLUMN source INTEGER DEFAULT 2 NOT NULL;
     </sql>
   </revision>
+  <revision version="30" min_compatible="3">
+    <description>
+      Tables for Rekordbox library feature
+    </description>
+    <sql>
+     CREATE TABLE IF NOT EXISTS rekordbox_library (
+        id INTEGER primary key AUTOINCREMENT,
+        rb_id integer,
+        artist varchar(48), title varchar(48),
```

Comment:
Are those varchar size actual rekordbox limits, or did you just copy the traktor schema? If not please remove them and simply use TEXT.